### ISSUE TITLE
vcfwave not found

### ISSUE THREAD

**Issue by STF003** [vcflib/vcflib#388]
Hi vcflib-team:

I downloaded vcflib from Conda, and I don't found vcfwave in my path. What should I do? Thanks.

**Comment by etnite** [comment]
The version of vcflib on bioconda appears to be 1.0.3, which is unfortunately one version prior to the addition of vcfwave and vcfcreatemulti.

I've been trying to install a version of vcflib containing vcfwave/vcfcreatemulti on a machine running Ubuntu 18.04 and it's been quite challenging. The version available to me via apt-get is, I believe, v1.0.0.

Trying to build from source, I first had to build a newer version of CMake (3.26.4). However, partway through building I get the following error:

```
$ cmake -DCMAKE_BUILD_TYPE=Debug -DZIG=OFF -DOPENMP=OFF ..
-- The C compiler identification is GNU 11.1.0
-- The CXX compiler identification is GNU 11.1.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.29.1")
-- Could NOT find pybind11 (missing: pybind11_DIR)
-- Found BZip2: /usr/lib/x86_64-linux-gnu/libbz2.so (found version "1.0.6")
-- Looking for BZ2_bzCompressInit
-- Looking for BZ2_bzCompressInit - found
-- Looking for lzma_auto_decoder in /usr/lib/x86_64-linux-gnu/liblzma.so
-- Looking for lzma_auto_decoder in /usr/lib/x86_64-linux-gnu/liblzma.so - found
-- Looking for lzma_easy_encoder in /usr/lib/x86_64-linux-gnu/liblzma.so
-- Looking for lzma_easy_encoder in /usr/lib/x86_64-linux-gnu/liblzma.so - found
-- Looking for lzma_lzma_preset in /usr/lib/x86_64-linux-gnu/liblzma.so
-- Looking for lzma_lzma_preset in /usr/lib/x86_64-linux-gnu/liblzma.so - found
-- Found LibLZMA: /usr/lib/x86_64-linux-gnu/liblzma.so (found version "5.2.2")
-- Found ZLIB: /usr/lib/x86_64-linux-gnu/libz.so (found version "1.2.11")
-- Found CURL: /usr/local/lib/libcurl.so (found version "7.82.0")
--

-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE
-- Checking for module 'htslib'
--   Found htslib, version 1.10.2-117-ga79009b
-- Checking for module 'tabixpp'
--   No package 'tabixpp' found
--

-- WFA using include contrib/WFA2-lib
CMake Error at CMakeLists.txt:459 (pybind11_add_module):
  Unknown CMake command "pybind11_add_module".


-- Configuring incomplete, errors occurred!
```

**Comment by juliawiggeshoff** [comment]
@etnite you need to `pip install pybind11[global]`
